packages/ember-metal/lib/mixin.ts

```diff
@@ -548,14 +548,6 @@ export default class Mixin {
     }
   }
 
-  static _apply() {
-    return (applyMixin as any)(...arguments);
-  }
-
-  static applyPartial(obj: any, ...args: any[]) {
```

Comment:
removed static `applyPartial` since it is undocumented does not have usage in wild,  and `applyMixin` could be used instead